### ISSUE TITLE
Turborepo deployments on vercel missing `routes-manifest.json`

### ISSUE THREAD

**Issue by goellner** [zeit/now#7265]
We are running into this error on 95% of our builds at the moment: 

```
Error: The file "/vercel/path0/apps/storefront/.next/routes-manifest.json" couldn't be found. 
This is normally caused by a misconfiguration in your project.
```

I tested with all turborepo versions from 1.0.19 to 1.0.24. There is no way for us to debug this as we can't see the filesystem.

We tested with a known good test repo, where we had working builds before christmas and a redeploy fails now too. So it seems something on vercel changed and has to be fixed on vercel.

EDIT: we have no build errors prior to this and we can build locally with the `npx turbo run build --scope=storefront --include-dependencies --no-deps` command just fine

**Comment by Boytobeaman** [comment]
just deploy https://nextjs-two-mu-87.vercel.app/ it works fine

**Comment by goellner** [comment]
sorry, forgot to close this issue. we found the culprit and it was a wrongly configured turborepo on our side. i will leave the answer here, if anyone runs into the same issue in the future

this is the response we got from vercel support/turborepo:

"They had a misconfigured pipeline that did not properly re-specify outputs of .next/**  in the storefront#build task in the pipeline. As stated in docs: package-tasks do not “inherit” pipeline configuration, There is also no need to manually define the dependent package-specifoc tasks when the relationship is purely topologic (i.e. dependency-first order). Instead, the default pipeline in the kitchen sink example works as expected."

removed this part of our config and everything was working as expected again.

![Image_2022-01-06_13-06-11](https://user-images.githubusercontent.com/3986748/148786470-a8e8b1c5-a7f0-400e-bf8c-edb716b71594.jpg)
